phoenix-core/src/main/java/org/apache/hadoop/hbase/ipc/controller/MetadataRpcController.java

```diff
@@ -36,7 +37,16 @@
 			.add(PhoenixDatabaseMetaData.SYSTEM_CATALOG_NAME)
 			.add(PhoenixDatabaseMetaData.SYSTEM_STATS_NAME)
 			.add(PhoenixDatabaseMetaData.SYSTEM_SEQUENCE_NAME)
-			.add(PhoenixDatabaseMetaData.SYSTEM_FUNCTION_NAME).build();
+			.add(PhoenixDatabaseMetaData.SYSTEM_FUNCTION_NAME)
+            .add(SchemaUtil.getPhysicalTableName(PhoenixDatabaseMetaData.SYSTEM_CATALOG_NAME_BYTES, true)
```

Comment:
yeah, namespace mapping for system tables will not always be true as it is controlled with phoenix.connection.mapSystemTablesToNamespace, 
but this is just the list of system tables, so it includes SYSTEM.CATAOG and SYSTEM:CATALOG both to set the priority for RPC accordingly. 
do you think, it could have any impact?